src/content/concepts/why-webpack

```diff
@@ -0,0 +1,55 @@
+---
+title: Why webpack
+sort: 7
+contributors:
+- debs-obrien
+---
+
+## Why webpack?
+
+To understand why you should use webpack lets do a recap of how we use JavaScript on the web. The origins of how we use JavaScript on the web. 
+
+JavaScript is just a script and there are 2 ways to use it in the browser, loaded as a script tag or written directly inside your html. The problems that this causes is that it doesn't scale. You either have too many scripts that you try to load from your script tags in the html and this causes a huge performance problem. There is only a certain amount of concurrent requests that can fetch data at a single time so breaking your application into a hundred JavaScript files and loading them in the browser is a huge performance problem. One big JS file results in unmaintainable scripts that causes problems in scope, size, readability, fragility and monolith files.
+
+
+## IIFE's - Immediately invoked function expressions
+
+IIFE's solve big problems with regards to scoping. Every file is treated as an IIFE which allows us to concatenate or safely combine files without concern of scope collision. Wow, we can now write 100 files and squish them together without concerns that we are going to knock out a bunch of variables and scopes.
```

Comment:
Again here, I would rephrase to _IIFEs solve scoping issues for large projects_.

The following should go more from the structure of the file to the benefit so users can understand better, for example:

> When script files are wrapped by an IIFE, you can safely concatenate...

Reply:
done